kratos/sources/kernel.cpp

```diff
@@ -97,14 +97,12 @@ void Kernel::PrintData(std::ostream& rOStream) const {
     rOStream << std::endl;
     rOStream << "Modelers:" << std::endl;
     KratosComponents<Modeler>().PrintData(rOStream);
-
+    rOStream << std::endl;
     rOStream << "Loaded applications:" << std::endl;
```

Comment:
```suggestion
    rOStream << "\nLoaded applications:" << std::endl;
```

Reply:
I'll leave it as it is to keep the way its done above.